src/Scaler/Services/ScalerMetadata.cs

```diff
@@ -39,8 +41,16 @@ private string LeaseAccountHost
         {
             get
             {
-                var builder = new DbConnectionStringBuilder { ConnectionString = this.LeaseConnection };
-                return new Uri((string)builder["AccountEndpoint"]).Host;
+                if(!string.IsNullOrEmpty(this.LeaseConnection))
```

Comment:
Do we support connection string based connections or not?

Reply:
this was a miss